lib/vsc/mympirun/mpi/intelmpi.py

```diff
@@ -129,7 +129,7 @@ def mpiexec_set_global_options(self):
 
         # this one also needs to be set at runtime
         self.mpiexec_global_options['I_MPI_MPD_TMPDIR'] = "/tmp"
```

Comment:
if `$TMP` or `$TMPDIR` or `$TEMP` are set, these should be used. Only use the hardcode `/tmp` when there is nothing else.